odoo/addons/test_new_api/models.py

```diff
@@ -452,6 +452,7 @@ class CompanyDependentAttribute(models.Model):
     _description = 'Test New API Company Attribute'
 
     company = fields.Many2one('test_new_api.company')
+    company_foo = fields.Char(related='company.foo')
```

Comment:
The field has to be specified as `readonly=False` ;). (Just tested locally and the test works fine after this change)

Reply:
Oh, it's just new default readonly=True value for related fields. Thank you!